content/chapters/io/lecture/slides/linux-files.md

```diff
@@ -0,0 +1,110 @@
+## Linux Files
```

Comment:
This seems to sudden. I suggest you first present the I/O interface, before moving to I/O devices.

I would first show how you would use the I/O before showing the lower parts.

Reply:
I went with your approach - explaining the interface without going into detail about `file descriptors` and `devices` feels more natural.